newIDE/app/src/EventsSheet/SearchPanel.js

```diff
@@ -53,6 +61,12 @@ export default class SearchPanel extends PureComponent<Props, State> {
     }
   };
 
+  makeSearchResultsDirty = () => {
+    this.setState({ searchResultsDirty: true }, () => {
+      this.callLaunchSearch();
```

Comment:
Why launching the search after making the search result dirty?

Reply:
If the events are changed, we also need to update the search, right? What this does is, whenever I change the events it updates (rather 're-searches') to get the correct results. Not sure, if its most optimal. But works fine.